website/docs/r/oracle_database_system.html.markdown

```diff
@@ -0,0 +1,174 @@
+
+* `ssh_public_keys` - (Required) The public key portion of one or more key pairs used for SSH access to the Database system. Changing this forces a new Database system to be created.
+
+* `zones` - (Required) Specifies a list of Database System zones. Changing this forces a new Database System to be created.
+
+---
+
+* `cluster_name` - (Optional) The cluster name for Exadata and 2-node RAC virtual machine Database systems. The cluster name must begin with an alphabetic character, and may contain hyphens (-). Underscores (_) are not permitted. The cluster name can be no longer than 11 characters and is not case sensitive. Changing this forces a new Database system to be created.
+
+* `database_system_options` - (Optional) One or more `database_system_options` blocks as defined below. Changing this forces a new resource to be created.
+
+* `disk_redundancy` - (Optional) The type of redundancy configured for the Database system.  Possible values are `High` and `Normal`. NORMAL is 2-way redundancy. HIGH is 3-way redundancy. Changing this forces a new Database system to be created.
+
+* `display_name` - (Optional) The user-friendly name for the Database system. Changing this forces a new Database system to be created. The name does not need to be unique.
+
+* `domain` - (Optional) The domain name for the Database system. Changing this forces a new Database system to be created.
+
+* `initial_data_storage_size_in_gb` - (Optional) Size in GB of the initial data volume that will be created and attached to a virtual machine Database system. You can scale up storage after provisioning, as needed. Note that the total storage size attached will be more than the amount you specify to allow for REDO/RECO space and software volume. Changing this forces a new Database system to be created.
+
+* `node_count` - (Optional) The number of nodes in the Database system. For RAC Database systems, the value is greater than 1. Changing this forces a new Database system to be created.
+
+* `pluggable_database_name` - (Optional) The name of the pluggable database. The name must begin with an alphabetic character and can contain a maximum of thirty alphanumeric characters. Special characters are not permitted. The pluggable database name should not be the same as the database name. Changing this forces a new Database system to be created.
+
+* `storage_volume_performance_mode` - (Optional) The block storage volume performance level. Valid values are Balanced and HighPerformance. See [Block Volume Performance](/Content/Block/Concepts/blockvolumeperformance.htm) for more information. Changing this forces a new Database system to be created.
+
+* `tags` - (Optional) A mapping of tags which should be assigned to the Database System.
+
+* `time_zone` - (Optional) The time zone of the Database system, e.g., UTC, to set the timeZone as UTC. Changing this forces a new Database System to be created.
```

Comment:
if it's specified as UTC, it should be named as `time_zone_in_utc`. ([ref](https://github.com/hashicorp/terraform-provider-azurerm/blob/main/contributing/topics/reference-naming.md#general-property-naming-conventions))